src/reducers/index.test.ts

```diff
@@ -71,7 +73,20 @@ it('setMatchesList', () => {
     setMatchesList: matchesList
   };
   let initialState = storeStateDefault;
-  const expectedState = { ...storeStateDefault, matchesList: matchesList };
+  let { matchIdToMatchState } = initialState;
```

Comment:
revert this test. This test shouldn't need any changes.

Please add another test where the state has a match in matchesList and matchIdToMatchState, and then we call setMatchesList with an empty list, so that should delete from both.

Reply:
for that test, should I reset the currentMatchIndex to like -1 or something? Or else the bounds check will fail

Reply:
Perfect, you found a bug :)
currentMatchIndex may need to be updated when dispatching setMatchesList
Yes, make sure we have a test that covers that.
Thanks!!